test/e2e/storage/vsphere/vsphere_volume_vsan_policy.go

```diff
@@ -341,3 +339,22 @@ func invokeStaleDummyVMTestWithStoragePolicy(client clientset.Interface, masterN
 	framework.ExpectNoError(err)
 	framework.ExpectEqual(isVMPresentFlag, false, errorMsg)
 }
+
+func getControlPlaneNode(client clientset.Interface) (string, error) {
+	regKubeScheduler := regexp.MustCompile("kube-scheduler-.*")
+	regKubeControllerManager := regexp.MustCompile("kube-controller-manager-.*")
+
+	podList, err := client.CoreV1().Pods(metav1.NamespaceSystem).List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		return "", err
+	}
+	if len(podList.Items) < 1 {
+		return "", fmt.Errorf("Can't find any pods in namespace %s to grab metrics from", metav1.NamespaceSystem)
```

Comment:
errors should start lowercase.
in this case the message should be different i guess? because we are not getting this node for metrics?

> "could not find kube-scheduler and kube-controller-manager pods in the %s namespace", ....

Reply:
OK, let me make error message lowercase.

> "could not find any control-plane pods in the %s namespace", ....

The above error case is for not getting any pods in the namespace before checking the pods are control plane.